lib/Doctrine/Common/Persistence/Mapping/StaticReflectionService.php

```diff
@@ -52,7 +52,7 @@ public function getClassNamespace($className)
     {
         $namespace = '';
         if (strpos($className, '\\') !== false) {
-            $namespace = strrev(substr( strrev($className), strpos(strrev($className), '\\')+1 ));
+            $namespace = strrev(substr(strrev($className), strpos(strrev($className), '\\')+1));
```

Comment:
Shouldn't there be spaces around `+`?

Reply:
It should indeed. We need to find the sniff and add it to `doctrine/coding-standard` then. Could you do it?

Reply:
There is Squiz.WhiteSpace.OperatorSpacing, but it wasn't compatible with `declare(strict_types=1)` and reports it as violation. Ended up with a hack to ignore declare in our private CS. :/